Source/WebKit/UIProcess/ios/WKModelView.mm

```diff
@@ -102,7 +102,7 @@ - (BOOL)createFileForModel:(WebCore::Model&)model
         return NO;
     }
 
-    String fileName = FileSystem::encodeForFileName(createVersion4UUIDString()) + ".usdz";
+    String fileName = makeString(FileSystem::encodeForFileName(createVersion4UUIDString()), ".usdz");
```

Comment:
I do not think we need to call `encodeForFileName` on a UUID. What character would we be encoding? I am pretty sure that lowercase letters, numeric digits, and hyphens don’t need encoding on any supported file system.

Reply:
```
   73 // The following lower-ASCII characters need escaping to be used in a filename
   74 // across all systems, including Windows:
   75 //     - Unprintable ASCII (00-1F)
   76 //     - Space             (20)
   77 //     - Double quote      (22)
   78 //     - Percent           (25) (escaped because it is our escape character)
   79 //     - Asterisk          (2A)
   80 //     - Slash             (2F)
   81 //     - Colon             (3A)
   82 //     - Less-than         (3C)
   83 //     - Greater-than      (3E)
   84 //     - Question Mark     (3F)
   85 //     - Backslash         (5C)
   86 //     - Pipe              (7C)
   87 //     - Delete            (7F)
   ```
   Yes, so, UUID strings (lower case alphabets + digits + hyphen) are not included, so we do not need `encodeForFileName` for UUID, removed.